parts/footer.html

```diff
@@ -0,0 +1,97 @@
+    <div class="wp-block-group dswp-align-right-desktop-left-mobile" style="margin-top:0;margin-bottom:0;padding-right:0;padding-left:0"><!-- wp:group {"style":{"spacing":{"padding":{"top":"0","bottom":"0"},"margin":{"top":"0","bottom":"0"}}},"layout":{"type":"constrained"}} -->
+    <div class="wp-block-group" style="margin-top:0;margin-bottom:0;padding-top:0;padding-bottom:0"><!-- wp:heading {"level":6,"style":{"spacing":{"margin":{"top":"0","bottom":"var:preset|spacing|30"}}}} -->
+    <h6 class="wp-block-heading" style="margin-top:0;margin-bottom:var(--wp--preset--spacing--30)">Organization</h6>
+    <!-- /wp:heading -->
+    
+    <!-- wp:paragraph {"style":{"typography":{"lineHeight":"3"}}} -->
+    <p style="line-height:3"><a href="https://www2.gov.bc.ca/" data-type="link" data-id="https://www2.gov.bc.ca/">TI Corp</a></p>
+    <!-- /wp:paragraph -->
+    
+    <!-- wp:paragraph {"style":{"typography":{"lineHeight":"3"}}} -->
+    <p style="line-height:3"><a href="https://www2.gov.bc.ca/gov/content/about-gov-bc-ca">Work at TI Corp</a></p>
+    <!-- /wp:paragraph -->
+    
+    <!-- wp:paragraph {"style":{"typography":{"lineHeight":"3"}}} -->
+    <p style="line-height:3"><a href="https://www2.gov.bc.ca/gov/content/home/disclaimer">Projects</a></p>
+    <!-- /wp:paragraph --></div>
+    <!-- /wp:group -->
+    
+    <!-- wp:group {"style":{"spacing":{"padding":{"top":"0","bottom":"0"},"margin":{"top":"0","bottom":"0"}}},"layout":{"type":"constrained"}} -->
+    <div class="wp-block-group" style="margin-top:0;margin-bottom:0;padding-top:0;padding-bottom:0"><!-- wp:heading {"level":6,"style":{"spacing":{"margin":{"top":"0","bottom":"var:preset|spacing|30"}}}} -->
+    <h6 class="wp-block-heading" style="margin-top:0;margin-bottom:var(--wp--preset--spacing--30)">Contact us</h6>
+    <!-- /wp:heading -->
+    
+    <!-- wp:paragraph {"style":{"typography":{"lineHeight":"3"}}} -->
+    <p style="line-height:3"><a href="https://www2.gov.bc.ca/" data-type="link" data-id="https://www2.gov.bc.ca/"></a><a href="https://www2.gov.bc.ca/" data-type="link" data-id="https://www2.gov.bc.ca/">Phone: 250-940-8516</a></p>
+    <!-- /wp:paragraph -->
+    
+    <!-- wp:paragraph {"style":{"typography":{"lineHeight":"3"}}} -->
+    <p style="line-height:3"><a href="https://www2.gov.bc.ca/gov/content/about-gov-bc-ca"></a><a href="https://www2.gov.bc.ca/gov/content/about-gov-bc-ca">Email: Admin@ticorp.ca</a></p>
+    <!-- /wp:paragraph -->
+    
+    <!-- wp:social-links {"iconColor":"ticorp-footer-background","iconColorValue":"#2D2D2D","size":"has-large-icon-size","className":"is-style-logos-only","style":{"spacing":{"blockGap":{"left":"var:preset|spacing|40"}}}} -->
+    <ul class="wp-block-social-links has-large-icon-size has-icon-color is-style-logos-only"><!-- wp:social-link {"url":"youtube.com","service":"youtube"} /-->
```

Comment:
are these social links supposed to actually have the urls of their accounts?

Reply:
Will updated when I the links, should have added empty links.